### PR TITLE
COG-1064-Add super-admin to roles

### PR DESCRIPTION
**Pull request by sunilb0575**
#### Which User story does this relate to (link)?
-https://osi-cwds.atlassian.net/browse/COG-1064

#### Brief feature description
-Add Super-admin to roles

### CODE REVIEW

**Comment by sunilb0575** [comment]
jenkins retest this please

File: app/javascript/selectors/userListSelector.js [hunk 1]
```diff
@@ -36,6 +36,11 @@ export const cardHeaderText = state => {
   const roles = safeGet(state, 'userList.adminAccountDetails.roles', [])
   const countyName = safeGet(state, 'userList.adminAccountDetails.county_name', '')
   const role = roles || []
-  const cardHeaderText = role.includes('State-admin') ? 'State Administrator View' : `County: ${countyName}`
-  return cardHeaderText
+  if (role.includes('Super-admin')) {
+    return 'Global Administrator view'
```
Comment (RamuKammagani-osi) [inline app/javascript/selectors/userListSelector.js]: is this right?? As per my understanding there is no UI Change at all, is it confirmed with PO? just checking.

**Comment by Sushmareddy-OSI** [comment]
Need to fix the codeClimate issue. Other than that, looks good!

File: app/javascript/_utils/checkAdminRoles.test.js [hunk 2]
```diff
@@ -9,6 +9,14 @@ describe('#getAdminOfficeIDs', () => {
     expect(getAdminOfficeIDs(account)).toEqual(['1234'])
   })
 
+  it('returns empty array when roles have Super-admin ', () => {
```
Comment (RamuKammagani-osi) [inline app/javascript/_utils/checkAdminRoles.test.js]: Can you also add 2 more scenarios when roles are null or undefined? I see you tested when account is undefined, But what if account exists and roles are undefined or null or some other text?

File: app/javascript/selectors/userListSelector.js (hunk 1 above)
Comment (sunilb0575) [reply app/javascript/selectors/userListSelector.js]: Yes, It is confirmed from PO.

**Comment by sunilb0575** [comment]
Sure. @RamuKammagani-osi @Sushmareddy-OSI I will create another PR for that code climate issue.
